lib/Installation/Warnings/WarningsController.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces business actions for Warning Popups
```

Comment:
I don't agree with using the word "business" everywhere. It can be confusing for people who are not so familiar with testing theory. Plus, if I google "business action" and "software testing" together, I don't get a clear definition.

Reply:
Thanks for the review and thanks for the point!;)

First of all, reading or adding new tests using [the approach](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/ui-framework-documentation.md), assumes a person got familiar with it, e.g. by reading our documentation where it is mentioned and described (Please, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/ui-framework-documentation.md#context section). 

Also, if you will google not just software testing but something like `"business actions" in automation testing`, you get description in several places.

If you think that we cover that part not enough in the documentation, feel free to raise that point as a ticket or update the documentation so that it will be more clear.

Reply:
As soon as we are using the term `business actions` all over the project, I guess this is not a topic of this particular PR. In case you see any better term, we can discuss this and plan activities to rename it everywhere.